libtenzir/src/exec_pipeline.cpp

```diff
@@ -246,6 +246,9 @@ auto exec_pipeline(std::string content,
         [&](diagnostic& d) {
           diag->emit(std::move(d));
         },
+        [&](std::string&, record&) {
+          // drop custom metrics.
```

Comment:
Treat similar to the handler below.

Reply:
I added custom metric handling in https://github.com/tenzir/tenzir/pull/4339/commits/26c37425433535201598f1644dbf473fdd7d27d4